src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/Workflow.kt

```diff
@@ -27,6 +33,10 @@ class Workflow(val manager: WorkflowManager, val id: String) {
 
     internal var toggleIntended: Boolean? = null
 
+    fun schedule(path: String, type: String = resourceType) {
+        WorkflowScheduler.execute(this, path, type)
```

Comment:
how about that WorkflowManager will have WorkflowScheduler so that you could access it 

`scheduler.execute(path, type)`